src/utils/http_helper.py

```
@@ -46,3 +46,21 @@ def http_get(endpoint: str, **kwargs) -> dict:
     return res.json()
+
+
+def http_delete(endpoint: str, **kwargs) -> dict:
+    """make a DELETE request"""
+    headers = kwargs.pop("headers", {})
+    timeout = kwargs.pop("timeout", None)
+
+    try:
+        res = requests.delete(endpoint, headers=headers, timeout=timeout, **kwargs)
+    except Exception as error:
+        logger.exception(error)
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)